game/scripts/npc/abilities/leshrac_split_earth.txt

```diff
@@ -34,7 +34,7 @@
     // Damage.
     //-------------------------------------------------------------------------------------------------------------
     "AbilityDamage"                                       "120 180 240 300 600 1200"
-    "AbilityDuration"                                     "2"
+    "AbilityDuration"                                     "1.5" //OAA
```

Comment:
make sure to change duration line 57 too.

Reply:
This change is just Max nudging me to rewrite Leshrac Split Earth.